CONTRIBUTING.md

```diff
@@ -38,6 +38,7 @@ cd playwright
 
 ```bash
 npm ci
+npx playwright install
```

Comment:
Just noticed that the same document also mentions this a little later, as part of the test docs: https://github.com/Skn0tt/playwright/blob/2fa6ad86a23ea4a4d23cab54b543b12ee26ce142/CONTRIBUTING.md#L156